<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>态势平台桌面客户端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1" rowspan="4">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>主机名：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editHostName">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>密  码：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLineEdit" name="editPassword">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>端  口：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editPort">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>数据库：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="editDatabase"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>用户名：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="editUser"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>表  名：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="editTable">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnConnect">
         <property name="text">
          <string>连接</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnShow">
         <property name="text">
          <string>显示选中表</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnAnalyze">
         <property name="text">
          <string>生成统计信息</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btnOutExcel">
         <property name="text">
          <string>导出到Excel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionadd"/>
    <addaction name="separator"/>
    <addaction name="actionqu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionadd">
   <property name="text">
    <string>添加水印</string>
   </property>
   <property name="iconText">
    <string>水印</string>
   </property>
  </action>
  <action name="actionqu">
   <property name="text">
    <string>取出水印</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
